START.htm

```diff
@@ -164,35 +164,26 @@ <h2 id="update">Update board with latest software</h2>
     <h3>Step #1: Download the latest software image</h3>
     <p>Download "Debian" from <a target="_blank" href="http://beagleboard.org/latest-images">http://beagleboard.org/latest-images</a>.</p>
     <p><small><b>Note:</b>&nbsp;Due to sizing necessities, this download may take 30 minutes or more.</small></p>
-    <p>The Debian distribution ships with the boards. The file you download will have an .img.xz extention. This is a compressed sector-by-sector image of the SD card.</p>
+    <p>The Debian distribution ships with the boards. The file you download will have an .img.xz extension. This is a compressed sector-by-sector image of the SD card.</p>
     <img src="Docs/images/download-latestimage.png" width="50%" />
 
-    <h3>Step #2: Install compression utility</h3>
-    <p>Download and install <a target="_blank" href="http://www.7-zip.org/download.html">7-zip</a>.</p>
-    <img src="Docs/images/download-7zip.png" width="50%" /><br />
-    <img src="Docs/images/run-7zip-installer.png" width="50%" /><br />
-
-    <h3>Step #3: Decompress the image</h3>
-    <p>Use 7-zip to decompress the SD card .img file</p>
-    <img src="Docs/images/extract-latestimage.png" width="50%" /><br />
-
-    <h3>Step #4: Install SD card programming utility</h3>
-    <p>Download and install <a target="_blank" href="https://launchpad.net/win32-image-writer/+download">Image Writer for Windows</a>. Be sure to download the binary distribution.</p>
-    <img src="Docs/images/download-win32diskimager.png" width="50%" /><br />
-    <img src="Docs/images/extract-win32diskimager.png" width="50%" /><br />
+    <h3>Step #2: Install SD card programming utility</h3>
+    <p>Download and install <a target="_blank" href="https://etcher.io/">Etcher</a>.</p>
+    <img src="Docs/images/download-etcher.png" width="50%" /><br />
+    <img src="Docs/images/install-etcher.png" width="50%" /><br />
     <p>Some general help on programming SD cards can be found on the <a target="_blank" href="http://help.ubuntu.com/community/Installation/FromImgFiles">Ubuntu Image Writer page</a>. 
 
-    <h3>Step #5: Connect SD card to your computer</h3>
+    <h3>Step #3: Connect SD card to your computer</h3>
     <p>Use the provided microSD card to SD adapter or a USB adapter to connect the SD card to your computer.</p>
 
-    <h3>Step #6: Write the image to your SD card</h3>
-    <p>Use either the Ubuntu Image Writer or instructions on its page to write the decompressed image to your SD card.</p>
-    <img src="Docs/images/write-latestimage.png" />
+    <h3>Step #4: Write the image to your SD card</h3>
+    <p>Use Etcher to write image to your SD card.</p>
```

Comment:
And I wonder if it'd be worth clarifying (just to remove any possible confusion) that Etcher will transparently decompress the image on-the-fly before writing it to the SD card?